modules/admin_manual/pages/maintenance/upgrading/manual_upgrade.adoc

```diff
@@ -1,11 +1,12 @@
 = Manual ownCloud Upgrade
 :toc: right
 :toclevels: 1
+:description: This document describes how to manually upgrade your ownCloud installation. Because installations can vary, this guide can only give an overview of methods and examples.
 :page-aliases: maintenance/manual_upgrade.adoc
 
 == Introduction
 
-This document describes how to manually upgrade your ownCloud installation. Because installations can vary, this guide can only give an overview of methods and examples. These examples need to be adapted according your needs and your environment.
+{description} These examples need to be adapted according your needs and your environment.
 
 NOTE: This guide assumes that you have basic knowledge about Unix terminology, commands and concepts. In case you do not feel familiar with this, ownCloud highly recommends that you create a playground first to try the steps and/or get, if you are entitled to, in touch with ownCloud Support to avoid breaking your system or data loss.
```

Comment:
```suggestion
NOTE: This guide assumes that you have basic knowledge about Unix terminology, commands and concepts. In case you are not familiar with these, ownCloud highly recommends that you create a playground first to try the steps and/or get in touch with ownCloud support to avoid breaking your system or losing data.
```